pkg/utils/timeutils.go

```diff
@@ -68,7 +77,7 @@ func Str2duration(duration string) (int64, error) {
 	return int64(i * multiply), nil
 }
 
-// convert time string to time (unix milisecond)
+// Str2unixTime converts time string to time (unix milisecond)
 func Str2unixTime(timeString string) (int64, error) {
```

Comment:
Also redundant.

Reply:
Comment was there. This is the standard documentation format in Go (start with the function name), see [here](https://blog.golang.org/godoc-documenting-go-code).

Reply:
Oh well.